### ISSUE TITLE
neard init should NOT use the same key-pair for validator-key and test.near access key

### ISSUE THREAD

**Issue by frol** [near/nearcore#2995]
**I believe this is somewhat blocked on #4347**

***Describe the bug***

Currently, `neard init` implementation generates a single key-pair and re-uses it for `validator_key.json` and `test.near` access key (listed in `genesis.json`). <span class="error">[We teach]</span>([https://github.com/near/nearup/issues/64#issuecomment-633654830](https://github.com/near/nearup/issues/64#issuecomment-633654830)) our users bad practices this way.

***To Reproduce***

```
 neard init
 ```

Observe the `~/.near/validator_key.json` and `test.near` account access key in `genesis.json` have the same `public_key` meaning that we recommend users of the localnet to use `validator_key.json` to do interactions with `near-shell`.

***Expected behavior***

`neard init` should generate a separate key-pairs, meaning that the access key-pair for `test.near` should be saved in `~/.near/test.near_key.json`.

Alternatively, we should improve the whole neard and near-shell story around keys management.

I assign @chefsale as he has already been leading some CLI improvements of neard CLI as part of [https://github.com/near/nearup/issues/81](https://github.com/near/nearup/issues/81).

/cc @abacabadabacaba @amgando @ailisp @ilblackdragon @bowenwang1996 @mfornet

**Comment by chefsale** [comment]
So currently if you don't specify the chain-id to the arguments it generates an localnet, if you specify it it generates "production keys" for the specified network. Obviously there is no difference between the keys, but we only allow one set of keys for one environment if you want to init the setup you will have to delete the current keys aka do the clean, so one thing we could do is just to name the localnet keys differently as users will still have to clean and init the network again to make it work, from my understanding that's how it kind of works now, not sure what is the exact key management method we are trying to introduce, as well what are the other places we are doing an similiar thing of not having two keys. cc: @bowenwang1996 @frol @ailisp

**Comment by bowenwang1996** [comment]
yeah if you specify chain id it will be fine. For localnet I think convenience matters a bit more.

**Comment by chefsale** [comment]
Yep, I agree with bowen, it seems introducing this would make it more complicated for users. So, not really sure what would be the best follow up on this issue.

**Comment by bowenwang1996** [comment]
Seems like we don't have an issue here.

**Comment by frol** [comment]
`validator_key.json` should never be used to sign transactions, shouldn't it? If so, we should not teach the users bad practices of reusing the keys (ref #2833) /cc @abacabadabacaba

**Comment by bowenwang1996** [comment]
Good point. To fix this we should generate separate key in `~/.near`, like `signer_key` or `account_key`. To make it work smoothly with near-shell is another endeavor on its own. Nearcore should not be aware of cli's like near-shell.
